app/gui2/src/components/HelpScreen.vue

```diff
@@ -0,0 +1,216 @@
+<script setup lang="ts">
+import { baseConfig, type ApplicationConfig, type Group, type Option } from '@/util/config'
+import { computed } from 'vue'
+
+const props = defineProps<{
+  unrecognizedOptions: string[]
+  appConfig: ApplicationConfig
+}>()
+
+export interface HelpScreenSection {
+  description?: string | undefined
+  name: string
+  entries: HelpScreenEntry[]
+}
+
+export interface HelpScreenEntry {
+  name: string
+  values: string[]
+  href?: string | undefined
+}
```

Comment:
Is there a reason why this specifically isn't in nearby `types.ts` file?

Reply:
nice catch... lol they *are* actually already in `types.ts`